### PR TITLE
Uses fat jars instead of exploded directories

### PR DESCRIPTION
**Pull request by adriancole**

### CODE REVIEW

**Comment by jamescway** [comment]
:+1:

File: web/run.sh [hunk 1]
```diff
@@ -11,4 +11,4 @@ DEFAULT_ROOTURL=http://localhost:8080/
 ROOTURL="-zipkin.web.rootUrl=${ROOTURL:-DEFAULT_ROOTURL}"
 
 echo "** Starting ${SERVICE_NAME}..."
-./$SERVICE_NAME/build/install/$SERVICE_NAME/bin/$SERVICE_NAME -zipkin.web.query.dest=${QUERY_ADDR} $ROOTURL
+java -jar ./$SERVICE_NAME/libs/${SERVICE_NAME}*-all.jar -zipkin.web.query.dest=${QUERY_ADDR} $ROOTURL
```
Comment (jamescway) [inline web/run.sh]: ${ROOTURL}?

File: query/run.sh [hunk 2]
```diff
@@ -26,7 +26,7 @@ else
 fi
 
 SERVICE_NAME="zipkin-query-service"
-CONFIG="${SERVICE_NAME}/config/query-cassandra.scala"
+CONFIG="/query-cassandra.scala"
 
 echo "** Starting ${SERVICE_NAME}..."
-./$SERVICE_NAME/build/install/$SERVICE_NAME/bin/$SERVICE_NAME -f $CONFIG
+java -jar ./$SERVICE_NAME/libs/${SERVICE_NAME}*-all.jar -f $CONFIG
```
Comment (jamescway) [inline query/run.sh]: debatable nit ${CONFIG}

File: web/run.sh (hunk 1 above)
Comment (adriancole) [inline web/run.sh]: sure np

**Comment by adriancole** [comment]
rebuilding the images and testing again

**Comment by jamescway** [comment]
sorry :cry:  should have said optional

**Comment by adriancole** [comment]
NP it works!

**Comment by adriancole** [comment]
@niccaluim fyi this changes some things

**Pull request merged**